src/Autofac.Pooling/AutofacPooledObjectPolicy.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Autofac.Core;
+using Microsoft.Extensions.ObjectPool;
+
+namespace Autofac.Pooling
+{
+    /// <summary>
+    /// Provides the <see cref="IPooledObjectPolicy{TLimit}"/> needed for creating/returning objects in the pool in an Autofac way.
+    /// </summary>
+    /// <typeparam name="TLimit">The object pool type.</typeparam>
```

Comment:
This is more like... the type of object _being pooled_, right? Not actually the type _of pool_.

Reply:
Fixed these (also renamed the type param name in a couple of places).